lib/content-services/src/lib/search/components/search-check-list/search-check-list.component.ts

```diff
@@ -77,6 +78,23 @@ export class SearchCheckListComponent implements SearchWidget, OnInit {
                 this.context.queryFragments[this.id] = '';
             }
         }
+        this.context.populateFilters
+            .asObservable()
+            .pipe(first())
+            .subscribe((filtersQueries) => {
+                if (filtersQueries[this.id]) {
+                    filtersQueries[this.id].forEach((value) => {
+                        const foundIndex = this.options.items.findIndex((option) => option.value === value);
```

Comment:
Instead of findIndex you can use find function which will return object on which you can set directly "checked" property without searching for this again using index.